charts/minecraft-proxy/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: minecraft-proxy
-version: 3.7.0
+version: 4.0.0
```

This looks good but I don't know if the nature of the change requires a major version bump.
 
```suggestion
version: 3.8.0
```